sentry_sdk/integrations/django/__init__.py

```diff
@@ -646,8 +646,9 @@ def connect(self):
     ignore_logger("django.db.backends")
 
 
-# https://github.com/django/django/blob/6a0dc2176f4ebf907e124d433411e52bba39a28e/django/db/backends/base/base.py#L29
-# Avaliable in Django 1.8+
-def _set_db_system_on_span(span, vendor):
-    # type: (Span, str) -> None
+def _set_db_data(span, vendor, connection_params):
+    # type: (Span, str, Dict[str, str]) -> None
     span.set_data(SPANDATA.DB_SYSTEM, vendor)
+    span.set_data(SPANDATA.DB_NAME, connection_params.get("database"))
+    span.set_data(SPANDATA.SERVER_ADDRESS, connection_params.get("host"))
+    span.set_data(SPANDATA.SERVER_PORT, connection_params.get("port"))
```

Comment:
These might also differ in different backends I think. [MySQL also sometimes sets](https://github.com/django/django/blob/89c8c2e8311bd31b79c0829055d1e494ab88b801/django/db/backends/mysql/base.py#L219C21-L219C32) the `unix_socket`, which we should probably then set on the span as `SPANDATA.SERVER_SOCKET_ADDRESS`.

Reply:
done